custom-recipes/timeseries-preparation-decomposition/recipe.json

```diff
@@ -2,7 +2,7 @@
   "meta": {
     "label": "Timeseries decomposition",
     "description": "Decompose your timeseries into trend, seasonality and residuals.",
-    "icon": "icon-calendar-empty",
+    "icon": "icon-dku-timeseries-decomposition icon-DKU_timeseries_decomposition",
```

Comment:
What is `icon-DKU_timeseries_decomposition` ?
You should use `"icon": "icon-dku-timeseries-decomposition icon-calendar-empty"` so that old version of DSS still have an icon

Reply:
Ok, I did not know that when you specified two icons like this, once icon could replace the other one
I just copied the format of the other recipes

Reply:
Are you sure it is working? When I try to switch from the dip's branch `enhancement/v200-add-decomposition-icon` to `release/9.0`, the icon is no replaced with an empty calendar